Documentation.md

```diff
@@ -1216,19 +1216,12 @@ The new API is under development and documentation.
 
 | Domain | Name | Description |
 | --- | --- | --- |
-| conversations.domain | `conversation.chat.started` | Chat conversations only |
-| conversations.domain | `conversation.chat.closed` | Chat conversations only |
-| conversations.domain | `conversation.call.started` | Call conversations only |
-| conversations.domain | `conversation.call.closed` | Call conversations only |
-| conversations.domain | `conversation.video.started` | Video conversations only |
-| conversations.domain | `conversation.video.closed` | Video conversations only |
-| conversations.domain | `conversation.social.started` | Social conversations only |
-| conversations.domain | `conversation.social.closed` | Social conversations only |
-| conversations.domain | `conversation.*.closed` | All conversations |
-| users.domain | `user.created` | User created |
-| users.domain | `user.updated` | User information updated |
-| users.domain | `visitor.updated` | Visitor information updated |
-| users.domain | `satisfaction.filled` |
+| conversations.domain | `conversation.started` | All conversations |
+| conversations.domain | `conversation.closed` | All conversations |
+| conversations.domain | `visitor.updated` | Visitor information updated | |
+| users.domain | `user.created` | User created | |
+| users.domain | `user.updated` | User information updated | |
+| users.domain | `satisfaction.filled` | |
 | connectors.domain | `connector.installed ` |
 | connectors.domain | `connector.uninstalled` |
 
```

Comment:
Ajouter en in progress les events : 
connector.installed
connector.uninstall
conversation.started
user.loggedOut ou user.loggedout ou user.logout
user.loggedIn ou user.loggedin ou user.login